src/main/java/bio/terra/cli/app/utils/DockerClientWrapper.java

```diff
@@ -243,11 +243,12 @@ private RuntimeException wrapExceptionIfDockerConnectionFailed(RuntimeException
         ex.getCause() != null
             && ex.getCause() instanceof IOException
             && ex.getCause().getMessage() != null
-            && (ex.getCause().getMessage().contains("Connection refused")
+            && (ex.getCause().getMessage().toLowerCase().contains("connection refused")
                 || ex.getCause().getMessage().contains("native connect() failed"));
     if (isDockerConnectionFailed) {
       return new UserActionableException(
-          "Connecting to Docker daemon failed. Check that Docker is installed and running.", ex);
+          "Connecting to Docker daemon failed. Check that Docker is installed and running. "
+              + "To run apps without Docker, use the LOCAL_PROCESS app launch mode (terra config set app-launch LOCAL_PROCESS).", ex);
```

Comment:
The original plan was to put a check in the config set code for app-launch that would block the user from picking docker when docker wasn't available. I like this change better because it's less intrusive.